core/templates/dev/head/pages/landing/fractions/landing_page_teacher.html

```diff
@@ -147,14 +147,14 @@ <h2 class="oppia-fractions-landing-h2" style="color: #242424">Students are guide
           <div class="row">
             <div class="col-sm-6 oppia-fractions-landing-image-div">
               <video controls type="video/mp4" ng-src="<[getStaticVideoUrl('/fractions_landing/teachers/fractions_video.mp4')]>" class="oppia-fractions-landing-image
-              oppia-fractions-landing-image-desktop">Sorry, your browser doesn't support embedded videos.
+              oppia-fractions-landing-image-desktop" style="left: 10%">Sorry, your browser doesn't support embedded videos.
               </video>
               <video controls type="video/mp4" ng-src="<[getStaticVideoUrl('/fractions_landing/teachers/fractions_video.mp4')]>" class="oppia-fractions-landing-image
               oppia-fractions-landing-image-mobile">Sorry, your browser doesn't support embedded videos.
               </video>
             </div>
             <div class="col-sm-6">
-              <h1 class="oppia-fractions-landing-h1" style="color: #FFFFFF; text-align: right; padding-right: 80px; white-space: nowrap;">Easy-to-follow lessons</h1>
+              <h1 id="easyToFollow" class="oppia-fractions-landing-h1" style="color: #FFFFFF; text-align: right; padding-right: 80px;">Easy-to-follow lessons</h1>
```

Comment:
We don't use ids in general for CSS. Use a class instead, following the existing style.

Reply:
I'll definitely change it to class to comply with the coding style.